connect/add_ccloud_tenant.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+set -Eeuo pipefail
+
+if [ -z "${1:-}" ]; then
+    echo -e "\033[91mPlease, indicate tenant/realm!\033[0m"
+    exit 1
+fi
+
+source scripts/lib.sh || \
+    ( echo -e "\033[91mRun this script from root folder\033[0m" && \
+      exit 1 )
+
+# this can fail if a tenant already exists on the cluster with the same name
+# this shouldn't stop the local tenant creation process so we `|| true` it
+$GWM_RUN add_ccloud_tenant "$1" || true
```

Comment:
I think that it's better for the user, maintenance, and... whatever to simplify and reduce the number of files, options, cases...

I would suggest a different approach to this problem.

In the `connect/add_tenant.sh`

```bash
# headers and ...

set -Eeuo pipefail

if [ -z "${1:-}" ]; then
    echo -e "\033[91mPlease, indicate tenant/realm!\033[0m"
    exit 1
fi

source scripts/lib.sh || \
    ( echo -e "\033[91mRun this script from root folder\033[0m" && \
      exit 1 )

source options.txt

if [ "$AETHER_CONNECT_MODE" = 'LOCAL' ]; then
    # this can fail if a tenant already exists on the cluster with the same name
    # this shouldn't stop the local tenant creation process so we `|| true` it
    $GWM_RUN add_ccloud_tenant "$1" || true
fi

if [ "$AETHER_CONNECT_MODE" = 'CONFLUENT' ]; then
    $GWM_RUN add_kafka_tenant "$1"
fi
```

same comment for the rest of script files, setup, start, stop...